packages/components/src/components/date-picker/date-picker.scss

```diff
@@ -38,7 +38,10 @@
     }
     .dc-icon {
         top: 25%;
-        cursor: pointer;
+
+        &:not(.dc-icon--disabled) {
```

Comment:
```suggestion
        &:not(&--disabled) {
```

Reply:
I've test it, sorry this is not working.